.changeset/silver-jokes-study.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/dxfeed-secondary-adapter': patch
+---
+
+Add integration tests for dxfeed-secondary-adapter
```

Comment:
I don't think a changeset is necessary for adding/modifying tests only. It would matter if you had to change the exports to something, but I don't think it's necessary here

Reply:
Sounds good, just removed it